invenio_communities/members/services/service.py

```diff
@@ -420,14 +460,16 @@ def _members_search(
             extra_filter=filter,
             **kwargs
         )
-        search_result = search.execute()
+        search_result = search.scan() if scan else search.execute()
 
         return self.result_list(
             self,
             identity,
             search_result,
             params,
-            links_tpl=LinksTemplate(
+            links_tpl=None
+            if scan
+            else LinksTemplate(
```

Comment:
why do we need to return no links if we scan ?

Reply:
Links for each ResultItem will be created.
Links for the ResultList are not possible, as the pagination depends on the total hits of the query. When using scan, the hits object is only a generator and no information about total number of results is available (https://github.com/inveniosoftware/invenio-records-resources/blob/5335294dade21decea0f527022d96e12e1ffad52/invenio_records_resources/services/records/results.py#L165-L172)

The tests should cover dumping the result of scan and search, and checking if all dumped items are the same (including links)